sources/loc/strings.zh.resx

```diff
@@ -181,13 +181,13 @@
     <value>优化卡组时锁定此幻卡</value>
   </data>
   <data name="DeckCtrl_CtxMenu_PickCard" xml:space="preserve">
-    <value>选择幻卡：</value>
+    <value>选择幻卡:</value>
   </data>
   <data name="DeckCtrl_CtxMenu_UseOnlyOwned" xml:space="preserve">
     <value>仅使用已拥有的幻卡</value>
   </data>
   <data name="FavDeckCtrl_Edit" xml:space="preserve">
-    <value>编辑</value>
+    <value>⚙️</value>
```

Comment:
A special note, the phrase "编辑" is too long to be fully displayed in the button. I consider using Emoji "⚙️" to represent the text without zooming the icon.But I'm not sure if this will cause strange problems in some systems, what do you think?

Reply:
I think that text should be used, as it's more descriptive. Will scale that instance and start messing with changing UI to more automatic layouts.